docs/source/community/persons_of_interest.rst

```diff
@@ -311,6 +312,12 @@ PowerPC
 
 -  (emeritus) Alfredo Mendoza (`avmgithub <https://github.com/avmgithub>`__)
 
+x86 CPU
```

Comment:
Just want to clarify, that this category includes entire x86 architecture, regardless of the vendor.

Reply:
That's correct!